src/TMxpElementRegistry.h

```diff
@@ -33,6 +33,7 @@ struct TMxpElement
     QString name;
     QString definition;
     QStringList attrs;
+    QMap<QString, QString> defval;
```

Comment:
What does defval mean?

Reply:
These are the default values of attributes. Consider <!EL USER FORE=&f; ATT='f=red'> if <USER> is called w/o an 'f' attribute, f will default to red. In this case we would have defval["f"]="red" .

Do you have a suggestion for a better name? Should I add a comment to the source? Thx.

Reply:
Ah - `defaultValue` is good then